logrotate/CMakeLists.txt

```diff
@@ -21,12 +21,25 @@ SET(ACTIVE_CPP0xx_DIR "Release")
 INCLUDE (${LogRotate_SOURCE_DIR}/Options.cmake)
 INCLUDE_DIRECTORIES(${G3_HEADER_PATH})
 
+IF (MSVC)
+    SET(CMAKE_DEBUG_POSTFIX "_d")
+ENDIF()
+
 #Find all libraries
 
 find_package(Boost COMPONENTS filesystem system REQUIRED)
 include_directories(${Boost_INCLUDE_DIRS})
 
-find_library(G3LOG ${G3LOG_LIBRARY_TYPE} PATHS ${G3_LIBRARY_PATH})
+find_package( ZLIB REQUIRED )
+include_directories( ${ZLIB_INCLUDE_DIRS} )
+
+IF (MSVC)
+   find_library(G3LOG_DEBUG ${G3LOG_LIBRARY_TYPE}_d PATHS ${G3_LIBRARY_PATH_DEBUG})
+   find_library(G3LOG_RELEASE ${G3LOG_LIBRARY_TYPE} PATHS ${G3_LIBRARY_PATH_RELEASE})
```

Comment:
yes, please make a pull request on g3log as well and let's both of us test this together.  I have access to a Windows VM that I can test it on.